client/components/room/Video.jsx

```diff
@@ -56,13 +55,21 @@ class Video extends React.Component {
     this.hangup = this.hangup.bind(this);
     this.handleRemoteHangup = this.handleRemoteHangup.bind(this);
     this.stop = this.stop.bind(this);
+    this.requestTurn = this.requestTurn.bind(this);
     this.toggleVideo = this.toggleVideo.bind(this);
     this.toggleAudio = this.toggleAudio.bind(this);
+    this.handleCopy = this.handleCopy.bind(this);
   }
 
   componentDidMount() {
     let localVideo = document.getElementById('localVideo');
     let remoteVideo = document.getElementById('remoteVideo');
+    if (location.hostname !== 'localhost') {
+      this.requestTurn(
+        'http://numb.viagenie.ca?username=andy.yeo@gmail.com&key=hackreactor'
```

Comment:
It doesn't work, it's just part of the experiment phase. Sorry, the TURN stuff wasn't meant to get lumped to this branch but I guess it just happened.